bats/fb-katello-content.bats

```diff
@@ -77,6 +77,7 @@ setup() {
   fi
 
   tSkipIfOlderThan43
+  tSkipUnlessContentType 'ostree'
```

Comment:
I was wondering if it could be more efficient and only check it once, but I think `setup_file` isn't in some older bats versions we may test with.

Reply:
Given EL7 is dead, we could just stop installing bats via package (from epel) and always use git?

Reply:
I thought about the same thing. Probably a good thing for a new PR.